src/js/helpers/general.js

```diff
@@ -0,0 +1,2 @@
+// check if browser is IE
+export const isIe = () => !!document.documentMode;
```

Comment:
can we rename this to helper/browser.js?

Reply:
done